cpp/oneapi/dal/data/graph_service_functions.hpp

```diff
@@ -0,0 +1,84 @@
+/* file: graph_service_functions.hpp */
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+/*
+//++
+//  Graph types and service functionality
+//--
+*/
+
+#pragma once
+
+#include <cstdint>
```

Comment:
I still have the same question - should not cstdint be introduced once, for the whole library, and is used consitently by all components?

Reply:
It should. Will be fixed.